src/ai/backend/agent/server.py

```diff
@@ -556,6 +557,39 @@ async def commit(
             "path": str(Path(subdir, filename)),
         }
 
+    @rpc_function
+    @collect_error
+    async def get_local_config(self):
```

Comment:
Please add return type of this function. mypy won't check code inside function without return type.